<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Xview</class>
 <widget class="QMainWindow" name="Xview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1650</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xview</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushbuttonSelectFolder">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttonRefreshFolder">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>981</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_plotRaw"/>
   </widget>
   <widget class="QLabel" name="labelWorkingFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>321</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>61</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>620</y>
      <width>61</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_CurrentFolder_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numerator</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CurrentFolder_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>600</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Denominator</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxApplyLog">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>630</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Natural log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttonPlotSelectedFiles">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>710</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdge">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>30</y>
      <width>32</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>70</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>XANES</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdge_2">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>70</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdge_3">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>eV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>eV</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxInvert">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>650</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorkingFolder_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FileType</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFileType">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorkingFolder_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort by</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSortFilesBy">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>pushbuttonSelectFolder</zorder>
   <zorder>pushbuttonRefreshFolder</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>labelWorkingFolder</zorder>
   <zorder>listFiles</zorder>
   <zorder>listWidget</zorder>
   <zorder>listWidget_2</zorder>
   <zorder>label_CurrentFolder_2</zorder>
   <zorder>label_CurrentFolder_3</zorder>
   <zorder>checkBoxApplyLog</zorder>
   <zorder>pushbuttonPlotSelectedFiles</zorder>
   <zorder>lineEdge</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdge_2</zorder>
   <zorder>lineEdge_3</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>checkBoxInvert</zorder>
   <zorder>labelWorkingFolder_2</zorder>
   <zorder>comboBoxFileType</zorder>
   <zorder>labelWorkingFolder_3</zorder>
   <zorder>comboBoxSortFilesBy</zorder>
   <zorder>lineEdge_2</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdge_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>lineEdge</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
